src/pages/billing/NewBillingProjectWizard/GCPBillingProjectWizard/ContactAccountAdminToAddUserStep.ts

```diff
@@ -0,0 +1,50 @@
+import { div, h, span } from 'react-hyperscript-helpers'
+import { LabeledCheckbox } from 'src/components/common'
+import { Ajax } from 'src/libs/ajax'
+import Events from 'src/libs/events'
+import { styles } from 'src/pages/billing/NewBillingProjectWizard/GCPBillingProjectWizard/GCPBillingProjectWizard'
+import { ExternalLink } from 'src/pages/billing/NewBillingProjectWizard/StepWizard/ExternalLink'
+import { Step } from 'src/pages/billing/NewBillingProjectWizard/StepWizard/Step'
+import { StepFieldLegend, StepFields } from 'src/pages/billing/NewBillingProjectWizard/StepWizard/StepFields'
+import { StepHeader } from 'src/pages/billing/NewBillingProjectWizard/StepWizard/StepHeader'
+
+
+interface ContactAccountAdminToAddUserStepProps {
+  isActive: boolean
+  isFinished: boolean
+  verifiedUsersAdded: boolean | undefined
+  setVerifiedUsersAdded: (boolean) => void
+
+}
+
+export const ContactAccountAdminToAddUserStep = ({ isActive, ...props }: ContactAccountAdminToAddUserStepProps) => h(Step, { isActive }, [
+  StepHeader({ title: 'STEP 3' }),
+  h(StepFields, [
+    h(StepFieldLegend, { style: { width: '70%' } }, [
+      'Contact your billing account administrator and have them add you and ',
+      span({ style: { fontWeight: 'bold' } }, ['terra-billing@terra.bio']),
+      ' as a Billing Account User',
+      span({ style: { fontWeight: 'bold' } }, [' to your organization\'s billing account.'])
+    ]),
+    div({ style: { width: '30%', display: 'flex', flexDirection: 'row' } }, [
+      h(LabeledCheckbox, {
```

Comment:
There is something not correct about how the state persistence is working. On production, if I check this checkbox and then refresh the page, it remembers that I have checked the checkbox and that I am on Step 4. On your branch, after I refresh the checkbox is cleared.

Reply:
I had disabled the state storage so I could actually test behavior - when I put it back, the variable name had changed.

On a related note: should we clear all of this user state when they finish the last step? 
I found it really obnoxious that I'd come back and it would still have all the previous state from the last project.

Then again, the current implementation is really only intended to walk them through the first project. So maybe that's just something to revisit when we switch to using this instead of the modal.

Reply:
Right, with the current implementation you only see the wizard if you don't have any billing projects.